Taviloglu.Wrike.ApiClient/FoldersAndProjects/WrikeClient.FoldersAndProjects.cs

```diff
@@ -157,6 +157,18 @@ async Task<WrikeFolder> IWrikeFoldersAndProjectsClient.UpdateAsync(WrikeClientId
             return GetReponseDataFirstItem(response);
         }
 
+        async Task<IEnumerable<WrikeFolder>> IWrikeFoldersAndProjectsClient.UpdateFoldersAsync(WrikeClientIdListParameter folderIds, List<WrikeCustomFieldData> customFields, List<string> optionalFields)
+        {
```

Comment:
We should update WrikeFolder.OptionalFields to include all the values below
- metadata	Folder metadata entries
- hasAttachments	Has attachments
- attachmentCount	Attachment count
- description	Description
- briefDescription	Brief description
- customFields	Custom fields
- customColumnIds	Associated custom field IDs
- superParentIds	List of super parent folder IDs (applicable to 'Selective Sharing' labs feature)
- space	Is folder a space
- contractType	Contract type

and **allow only contractType** to be passed as parameter. Please see [GetFoldersAsync](https://github.com/staviloglu/Taviloglu.Wrike.ApiClient/blob/e42dda18a50a6c713df50d1ed3a9afc0907669fc/Taviloglu.Wrike.ApiClient/FoldersAndProjects/WrikeClient.FoldersAndProjects.cs#L25) which can be taken as sample for optionalFields validation